src/deck-options.md

```diff
@@ -411,12 +414,126 @@ to forget, it can help you remember more material in the same amount of time.
 This setting is shared by all deck presets.
 
 FSRS is currently in the advanced section, as it was only just integrated into
-Anki in the 23.10 release. When you enable the setting, some new options will
-become available, and SM-2 specific settings will be hidden.
+Anki in the 23.10 release. When you enable the setting, some new options will 
+become available, and SM-2 specific settings, such as "Graduating interval", 
+"Easy bonus", etc, will be hidden. This is because these settings are irrelevant
+when FSRS is enabled.
 
 If you previously used the 'custom scheduling' version of FSRS, please make
 sure you clear out the custom scheduling section before enabling FSRS.
 
+
+
+- **Maximum interval.**
+This works in the same way as the default algorithm: it is the maximum number
+of days a review card will wait. For more info, please see [next section](#maximum-interval)
+
+- **Desired Retention.**
+The most important setting to configure is the desired retention:this will
+determine the fraction of cards recalled successfully when they are due. 
```

Comment:
Adding the graph from that page you linked that shows red bars on the left and right would help users to visualise what's going on here. 

I suggest rewriting to the following:

*Desired retention* controls how likely you are to remember cards when they are reviewed. The default value of 0.9 will schedule cards so you have a 90% chance of remembering them when they come up for review again.

Here is a graph that shows how adjusting this value will affect your workload:

[insert graph]

There are two things to notice:

- As desired retention approaches 1.0, the frequency that you need to review cards increases drastically. For example, imagine you have a card that you have a 90% chance of remembering after 100 days. If your desired retention was 0.95, you'd need to review it after 47 days instead (approximately twice as frequently). At 0.97, the delay would be only 27 days (approximately 3.7x as frequently). At 0.99, you'd be reviewing every 9 days (more than 10x what you'd be doing with the defaults).
- As desired retention decreases, you'll forget a greater percentage of your cards, and those cards will need to be reviewed again. Eventually, you'll get to a point where the forgotten cards contribute more to your workload than you gain from the longer delays, which is why you see the workload on the left of the graph increasing. Also, bear in mind that forgetting material frequently is demotivating.

For these reasons, we suggest you be conservative when adjusting this number, and recommend you keep it between 0.85 and 0.95.

Reply:
added, thanks!